site2/docs/admin-api-persistent-topics.md

```diff
@@ -117,137 +112,132 @@ $ pulsar-admin persistent revoke-permission \
+  -   **msgRateOut**: The sum of all local and replication consumers' dispatch rates in messages per second.
 
-  -   **msgThroughputOut**: Same as above, but in bytes per second instead of messages per second
+  -   **msgThroughputOut**: The sum of all local and replication consumers' dispatch rates in bytes per second.
 
-  -   **averageMsgSize**: The average size in bytes of messages published within the last interval
+  -   **averageMsgSize**: The average size(in bytes) of messages published within the last interval.
 
-  -   **storageSize**: The sum of the ledgers' storage size for this topic. Space used to store the messages for the topic
+  -   **storageSize**: The sum of the ledgers' storage size for this topic. The space used to store the messages for the topic.
 
-  -   **publishers**: The list of all local publishers into the topic. There can be zero or thousands
+  -   **publishers**: The list of all local publishers into the topic. The list ranges from zero to thousands.
 
-      -   **msgRateIn**: Total rate of messages published by this publisher in messages per second 
+      -   **msgRateIn**: The total rate of messages published by this publisher in messages per second.
 
-      -   **msgThroughputIn**: Total throughput of the messages published by this publisher in bytes per second
+      -   **msgThroughputIn**: The total throughput of the messages published by this publisher in bytes per second.
 
-      -   **averageMsgSize**: Average message size in bytes from this publisher within the last interval
+      -   **averageMsgSize**: The average message size in bytes from this publisher within the last interval.
 
-      -   **producerId**: Internal identifier for this producer on this topic
+      -   **producerId**: The internal identifier for this producer on this topic.
 
-      -   **producerName**: Internal identifier for this producer, generated by the client library
+      -   **producerName**: The internal identifier for this producer, generated by the client library.
 
-      -   **address**: IP address and source port for the connection of this producer
+      -   **address**: The IP address and source port for the connection of this producer.
 
-      -   **connectedSince**: Timestamp this producer was created or last reconnected
+      -   **connectedSince**: The timestamp when this producer is created or reconnected last time.
 
-  -   **subscriptions**: The list of all local subscriptions to the topic
+  -   **subscriptions**: The list of all local subscriptions to the topic.
 
-      -   **my-subscription**: The name of this subscription (client defined)
+      -   **my-subscription**: The name of this subscription. It is defined by the client.
 
-          -   **msgRateOut**: Total rate of messages delivered on this subscription (msg/s)
+          -   **msgRateOut**: The total rate of messages delivered on this subscription (msg/s).
```

Comment:
use the same expression: use "msg/s" and (bytes/s) or in use "in messages per second" "in bytes per second"